issuer-front-end/src/CreateSchemaAndMetadataFiles.tsx

```diff
@@ -360,17 +370,13 @@ export default function CreateSchemaAndMetadataFiles() {
                             <option value="date-time">DateTime</option>
                         </select>
                     </label>
-                    <br />
-                    <br />
-                    <div>
-                        <input
-                            type="checkbox"
-                            id="checkBox"
-                            name="checkBox"
-                            onChange={(event) => changeCheckBox(required, event)}
-                        />
-                        <label htmlFor="checkBox">&nbsp;Is Type Required</label>
-                    </div>
+                    <label htmlFor="checkBox">&nbsp;Attribute is Required&nbsp;</label>
```

Comment:
If the intention was to read this in one sentence since the dropdown and text are in one line now.

E.g.
`Date-time` attribute is required.

Alternatively, add a breakpoint here.

```suggestion
                    <label htmlFor="checkBox">&nbsp;attribute is required.&nbsp;</label>
```